docs/source/install/upgrades.rst

```diff
@@ -50,18 +50,9 @@ Upgrade Procedure
 
 .. sourcecode:: bash
 
-  # Stop related services
-  service mistral-api stop
-  service mistral stop
-
-  # Upgrade database
   /opt/stackstorm/mistral/bin/mistral-db-manage --config-file /etc/mistral/mistral.conf upgrade head
```

Comment:
Hm so messing up with the database schema doesn't affect running services aka restart is not needed?